src/boot/StoreProvider.js

```diff
@@ -1,24 +1,25 @@
 /* @flow */
 import React, { PureComponent } from 'react';
 import { Provider } from 'react-redux';
+import { PersistGate } from 'redux-persist/lib/integration/react';
 
 import type { ChildrenArray } from '../types';
-import store, { restore } from './store';
+import store, { persistor } from './store';
 import timing from '../utils/timing';
+import LoadingScreen from '../start/LoadingScreen';
 
 type Props = {
   children: ChildrenArray<*>,
 };
 
 export default class StoreHydrator extends PureComponent<Props> {
-  componentDidMount() {
```

Comment:
We would want to keep this metric.
I guess we need to implement it differently.

Reply:
Looking at this:
https://github.com/rt2zz/redux-persist/blob/c9ae1903c3c5dea8604dd79ceea50c8a0a64f8fd/src/integration/react.js#L38
`onBeforeLoad` is probably a good place to start the timing, but stopping the timing is not so clearly defined... zi would have hoped for a `onAfterLoad`.
We can do it on unmounting of the `loading` component.